modin/pandas/test/test_io.py

```diff
@@ -95,6 +96,33 @@ def setup_clipboard(row_size=NROWS):
     df.to_clipboard()
 
 
+def parquet_eval_to_file(modin_obj, pandas_obj, fn, extension, **fn_kwargs):
+    """Helper function to test `to_parquet` method.
```

Comment:
The docstring is not numpydoc-like style yet. Please, fix it. You can look at how you document `to_parquet` as an example. https://github.com/modin-project/modin/pull/3573/files#diff-d6ab859ea39dde23e5d8b784b25150c9cf1b52b7f7d9233b99fd355145127130R228

Reply:
We have `eval_to_file` to check `to_*` functions. Why do we need this specific function for `to_parquet`?

Reply:
Because modin to_parquet implementation creates multiple `.parquet` files (one for each partition) that needs to be checked against pandas which creates a single `.parquet` file. The other `to_*` modin implementations create a single file that can be checked against pandas, but for `to_parquet` we have to have specific code to test reading and writing entire directories and checking them against a single. Additionally, tearing these directories/files down at the end of the test requires specific code not handled by `eval_to_file`.

Reply:
makes sense

Reply:
@devin-petersohn @YarShev if the behavior of the Modin function differs from the behavior of the pandas function, we should add it to the experimental mode, just as it was with `read_pickle_distributed` and `to_pickle_distributed` functions. Why is it done differ in this case?

Reply:
This is still consistent with parquet format, where `read_pickle_distributed` is not consistent with the format. We can still use pandas to read the output of this `parquet` writer, but it wouldn't work to read the directory of pickle files created by `read_pickle_distributed`.

Reply:
got it, thank you